DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseDetails.cshtml

```diff
@@ -0,0 +1,192 @@
+          <label class="nhsuk-heading-s nhsuk-u-font-weight-normal nhsuk-u-margin-bottom-0">
+            Course name
+          </label>
+          <div class="nhsuk-u-margin-top-2">
+            @Model.CustomisationName -
+          </div>
+        </div>
+        <div class="nhsuk-grid-column-one-half nhsuk-u-margin-top-2">
+          <vc:text-input asp-for="@nameof(Model.CustomisationNameSuffix)"
+                         id="CustomisationNameSuffix"
+                         label="Add on to course name (optional)"
+                         populate-with-current-value="true"
+                         type="text"
+                         spell-check="true"
+                         autocomplete=""
+                         hint-text=""
+                         css-class="nhsuk-u-width-three-quarters nhsuk-u-font-weight-normal" />
+        </div>
+      </div>
+
+      <div class="nhsuk-form-group nhsuk-u-margin-bottom-5">
+        <fieldset class="nhsuk-fieldset">
+          <div class="nhsuk-checkboxes nhsuk-checkboxes--conditional">
+
+            <div class="nhsuk-u-margin-bottom-3">
+              <div class="nhsuk-checkboxes__item">
+                <input class="nhsuk-checkboxes__input"
+                       id="PasswordProtected"
+                       name="PasswordProtected"
+                       asp-for="PasswordProtected"
+                       type="checkbox"
+                       aria-controls="@passwordId"
+                       aria-expanded="@(Model.PasswordProtected ? "true" : "false")">
+                <label class="nhsuk-label nhsuk-checkboxes__label" for="PasswordProtected">
+                  Password protected
+                </label>
+              </div>
+
+              <div class="@passwordCss" id="@passwordId">
+                <div class="nhsuk-form-group">
+                  <vc:text-input asp-for="@nameof(Model.Password)"
+                                 label="Enter the password for the course"
+                                 populate-with-current-value="true"
+                                 type="text"
+                                 spell-check="false"
+                                 autocomplete=""
+                                 hint-text=""
+                                 css-class="nhsuk-input nhsuk-u-width-one-half" />
+                </div>
+              </div>
+            </div>
+
+            <div class="nhsuk-checkboxes__item">
+              <input class="nhsuk-checkboxes__input"
+                     id="ReceiveNotificationEmails"
+                     name="ReceiveNotificationEmails"
+                     asp-for="ReceiveNotificationEmails"
+                     type="checkbox"
+                     aria-controls="@emailId"
+                     aria-expanded="@(Model.ReceiveNotificationEmails ? "true" : "false")">
+              <label class="nhsuk-label nhsuk-checkboxes__label" for="ReceiveNotificationEmails">

```

Comment:
Same for this one:
If you change this to asp-for it will get rid of the cannot resolve id error that you will get by changing the input above to just asp-for

Reply:
changed